src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,122 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
We recommend using SEQUENCE as a generation type strategy because Postgres uses an incremental id common to all tables :)